packages/components/src/input-control/input-field.tsx

```diff
@@ -168,9 +168,6 @@ function InputField(
 	const dragGestureProps = useDrag< PointerEvent< HTMLInputElement > >(
 		( dragProps ) => {
 			const { distance, dragging, event } = dragProps;
-			// The event is persisted to prevent errors in components using this
-			// to check if a modifier key was held while dragging.
-			event.persist();
```

Comment:
Was this related to the fix?

Shift-dragging still works as expected, but not sure what the original problem was. (Checking with @aaronrobertshaw from the git blame.)

Reply:
It's related in that I noticed console errors after the change to avoid pointer capture. The errors were about `persist` not existing on `event`. At first, I added optional-chaining 41f5161410afd290a1775fe0afe82aa3bb174754 to avoid those errors. Later, I remembered we're on React 17 [so using `persist` is outdated](https://reactjs.org/docs/legacy-event-pooling.html).

Reply:
From memory, this was the [original issue](https://github.com/WordPress/gutenberg/issues/30073) and was related to a need to check modifier keys being held when dragging on a UnitControl or BoxControl.

Reply:
Ah I see, thanks!